packages/aws-cdk-lib/aws-lambda-nodejs/lib/Dockerfile

```diff
@@ -39,4 +42,9 @@ RUN npm config --global set update-notifier false
 # create non root user and change allow execute command for non root user
 RUN /sbin/useradd -u 1000 user && chmod 711 /
 
+# Ensure all users can write to bun cache
+RUN mkdir /tmp/bun-cache && \
+    chmod -R 777 /tmp/bun-cache && \
+    echo -e "[install.cache]\ndir = \"/tmp/bun-cache\"" >> /home/user/.bunfig.toml
```

Comment:
There's no `bun config set --global` command for this cache location. The best place to set this is in the global `user` homedir according to [the docs](https://bun.sh/docs/install/cache).

Therefore, this cache statement needs to happen after `useradd` is called, not above where the other caches are configured. If it's called before, even if I create `/home/user`, `useradd` overwrites it.